app.js

```diff
@@ -20,18 +21,23 @@ const inputName = document.getElementById('given-user-name');
 const errorMsg = document.getElementById('error-msg');
 
 // Game elements
-const paper = document.getElementById('paper');
-const rock = document.getElementById('rock');
-const scissors = document.getElementById('scissors');
+const selectedPaper = document.getElementById('paper');
```

Comment:
`selected` and `chosen` means the same - something what user selected/chosen 👽  Those elements are for choosing an element.

imo - it should be: `paperElement` or even `paperButton` if it's a button element. 
And I would switch `chosenPaper` to `chosenPaperElement` or `selectedPaperElement`. 

Ofc, it should be applied to Rock and siccors as well.